vita3k/renderer/src/vulkan/context.cpp

```diff
@@ -125,7 +125,7 @@ void set_context(VKContext &context, MemState &mem, VKRenderTarget *rt, const Fe
     }
 
     SceGxmDepthStencilSurface *ds_surface_fin = &context.record.depth_stencil_surface;
-    if ((ds_surface_fin->depth_data.address() == 0) && (ds_surface_fin->stencil_data.address() == 0)) {
+    if (ds_surface_fin->depth_data.address() == 0 && ds_surface_fin->stencil_data.address() == 0) {
```

Comment:
Why remove this ? Is totaly good and correct like is it all continue like this request ()